src/Pages/BookMark/components/BookMarkLists.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import Room from "./Room";
+import "./BookMarkLists.scss";
+class BookMarkLists extends Component {
+  render() {
+    console.log("2번째", this.props.bookmarkList);
+    return (
+      <div className="book-mark-container">
+        <h2> 관심 장소 </h2>
+        <div className="book-mark-icons">
+          <i class="fas fa-hashtag"> </i> <i class="fas fa-plus"> </i>
+        </div>
+        <div className="book-mark-contents">
+          <ul className="slider-contents-lists">
+            {this.props.bookmarkList &&
+              this.props.bookmarkList.map((info) => {
+                return <Room info={info} />;
+              })}
```

Comment:
- 조건부 렌더링의 경우 조건에 배열 데이터 타입이 들어오는 경우에는 무조건 true 이기 때문에 적절하지 않네요-! 확인해주세요.
- 옵셔널 체이닝 검색해보시기 바랍니다.
```suggestion
            {
              this.props.bookmarkList?.map((info) => {
                return <Room info={info} />;
              })}
```

Reply:
아..배열의 경우는 생각 못 했네요! 그냥 값이 있으면 true 없으면 false라고 생각했어요. undefiend (= false) <-> true가 아니군요.